pkg/cloudprovider/aws/ami.go

```diff
@@ -83,12 +84,21 @@ func (p *AMIProvider) getAMIID(ctx context.Context, query string) (string, error
 	return ami, nil
 }
 
-func (p *AMIProvider) getSSMQuery(instanceType cloudprovider.InstanceType, version string) string {
+func (p *AMIProvider) getSSMQuery(ctx context.Context, amiFamily string, instanceType cloudprovider.InstanceType, version string) string {
 	var amiSuffix string
+	var arch string
 	if !instanceType.NvidiaGPUs().IsZero() || !instanceType.AWSNeurons().IsZero() {
 		amiSuffix = "-gpu"
 	} else if instanceType.Architecture() == v1alpha5.ArchitectureArm64 {
 		amiSuffix = "-arm64"
+		arch = "arm64"
+	} else if instanceType.Architecture() == v1alpha5.ArchitectureAmd64 {
+		arch = "arm64"
+	}
+
+	if amiFamily == v1alpha5.BottleRocket {
+		logging.FromContext(ctx).Debugf("AMIFamily was: %s", amiFamily)
+		return fmt.Sprintf("/aws/service/bottlerocket/aws-k8s-%s/%s/latest/image_id", version, arch)
```

Comment:
@suket22 It would be really nice if bottlerocket and eksoptimizedami were aligned to allow for simple substitution.

Reply:
I believe these SSM parameters are managed and owned by the [BottleRocket](https://github.com/bottlerocket-os/bottlerocket) folks - we could ask them if they'd like to restructure these but I don't think it's likely.